tests/profile/test_user_configurable_profiler_v3_batch_request.py

```diff
@@ -476,6 +475,8 @@ def test_build_suite_no_config(
     assert expectations_from_suite.issubset(possible_expectations_set)
     assert len(suite.expectations) == 48
 
+    # Note 20211209 - Profiler will also call ExpectationSuite's add_expectation(), but it will not
+    # send a usage_stats event when called from a Profiler.
```

Comment:
What about adding a `not in` assertion with the `add_expectation` event here and in `test_build_suite_with_config_and_no_semantic_types_dict()`?

Reply:
great idea. adding now